package.json

```diff
@@ -49,8 +49,7 @@
   "scripts": {
     "test": "lab -a code -t 100 -L -m 3000",
     "test-tap": "lab -a code -r tap -o tests.tap -m 3000",
-    "test-cov-html": "lab -a code -r html -o coverage.html -m 3000",
-    "postshrinkwrap": "json -I -f npm-shrinkwrap.json -e 'f(this) ; function f (p) { var deps = p.dependencies || {}; Object.keys(deps).forEach(function (n) { delete deps[n].resolved; f(deps[n]) }) }'"
```

Comment:
Ah, I see!  Very cool.  I was originally confused because the `json` package was added to devDeps but only used in the postshrinkwrap, which had disappeared.